modules/jupyter-widget/src/notebook-utils.js

```diff
@@ -0,0 +1,28 @@
+/* global document, window */
+/* Supports pydeck in a RequireJS-driven environment, like
+ * a Jupyter notebook or pydeck's standalone HTML page.
+ */
+import createDeckFromDependencies from './initializer';
+
+const REQUIREJS_URL = 'https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js';
+
+if (!window.require) {
+  const requirejs = document.createElement('script');
+  requirejs.setAttribute('src', REQUIREJS_URL);
+  document.head.appendChild(requirejs);
```

Comment:
Is this path tested? `require` won't be immediately available after attaching a `<script>` tag. Why not use the `addScript` utility from the other file?